admin.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href=""/>
+    <link rel="stylesheet" href="app.css">
+    <title>Admin</title>
+    </head>
+    <body>
```

Comment:
Same here. The body tag should have no space before it. Please remove the space before the opening tag and the closing tag.